language/de/acp/common.php

```diff
@@ -512,13 +512,13 @@
 	'LOG_ACL_ADD_ADMIN_GLOBAL_A_'		=> '<strong>Administratoren hinzugefügt oder geändert</strong><br />» %s',
 	'LOG_ACL_ADD_MOD_GLOBAL_M_'			=> '<strong>Globale Moderatoren hinzugefügt oder geändert</strong><br />» %s',
 
-	'LOG_ACL_ADD_USER_LOCAL_F_'			=> '<strong>Forenrechte eines Benutzers hinzugefügt oder geändert</strong> von %1$s<br />» %2$s',
-	'LOG_ACL_ADD_USER_LOCAL_M_'			=> '<strong>Moderationsrechte eines Benutzer für ein Forum hinzugefügt oder geändert</strong> von %1$s<br />» %2$s',
-	'LOG_ACL_ADD_GROUP_LOCAL_F_'		=> '<strong>Forenrechte einer Gruppe hinzugefügt oder geändert</strong> von %1$s<br />» %2$s',
-	'LOG_ACL_ADD_GROUP_LOCAL_M_'		=> '<strong>Moderationsrechte einer Gruppe für ein Forum hinzugefügt oder geändert</strong> von %1$s<br />» %2$s',
+	'LOG_ACL_ADD_USER_LOCAL_F_'			=> '<strong>Forenrechte eines Benutzers hinzugefügt oder geändert</strong> zu %1$s<br />» %2$s',
+	'LOG_ACL_ADD_USER_LOCAL_M_'			=> '<strong>Moderationsrechte eines Benutzer für ein Forum hinzugefügt oder geändert</strong> zu %1$s<br />» %2$s',
+	'LOG_ACL_ADD_GROUP_LOCAL_F_'		=> '<strong>Forenrechte einer Gruppe hinzugefügt oder geändert</strong> zu %1$s<br />» %2$s',
+	'LOG_ACL_ADD_GROUP_LOCAL_M_'		=> '<strong>Moderationsrechte einer Gruppe für ein Forum hinzugefügt oder geändert</strong> zu %1$s<br />» %2$s',
```

Comment:
Was wird denn da für %1 eingesetzt?

Reply:
Das sind Logeinträge für Foren / Moderatorrechte. Geschrieben z.B. [hier](https://github.com/phpbb/phpbb/blob/ec0423bf8260db49b231d0aa4ed33929a463fcb9/phpBB/includes/acp/acp_permissions.php#L974), das nutzt dann diese [Funktion](https://github.com/phpbb/phpbb/blob/ec0423bf8260db49b231d0aa4ed33929a463fcb9/phpBB/phpbb/log/log.php#L219).
Somit Daten: array($l_forum_list, $l_ug_list)
Also %1 Forenliste, %2 User/Gruppenliste
Da das jeweils Listen sind könnte da auch mehr als ein Forum/User/Gruppe stehen.

Reply:
Eine wirklich gute Lösung fällt mir da aber nicht ein, wie das sowohl im Singular als auch im Plural gut klingt...

Reply:
Wie wäre es mit "in"

Reply:
Wie wäre es so:
> >Forenrechte eines Benutzers hinzugefügt oder geändert</strong> in Support, Ankündigungen, News
» bei  Hans-Peter, Erwin, Newsschreiber

Reply:
Klingt ok.